web_src/js/components/RepoActionView.vue

```diff
@@ -335,6 +386,45 @@ const sfc = {
 
     isDone(status) {
       return ['success', 'skipped', 'failure', 'cancelled'].includes(status);
+    },
+
+    closeDropdown() {
+      if (this.menuVisible) this.menuVisible = false;
+    },
+
+    toggleTimeDisplay(type) {
+      if (type !== 'duration' && type !== 'stamp') return;
+      const otherType = 'durationstamp'.replace(type, '');
```

Comment:
It's very tricky, I think a clear if else it better.

Reply:
Done in https://github.com/go-gitea/gitea/pull/24876/commits/47d38f1a53ec485f7ef869124d37856c436e29d3